ui/lib/core/addon/components/radio-card.hbs

```diff
@@ -1,16 +1,16 @@
 <label
-  for={{dasherize @value}}
+  for={{dasherize @title}}
   class="radio-card {{if (eq @value @groupValue) 'is-selected'}} {{if @disabled 'is-disabled'}}"
   ...attributes
 >
   <ToolTip @verticalPosition="above" @horizontalPosition="center" as |T|>
-    <T.Trigger tabindex="-1">
+    <T.Trigger tabindex="-1" class="radio-card-container">
```

Comment:
I noticed that the bottom bar with radio button wasn't always pushed to the bottom of the container so I added a class to fix that